synapse/lib/layer.py

```diff
@@ -1197,7 +1204,7 @@ async def _layrV2toV3(self):
         self.meta.set('version', 3)
         self.layrvers = 3
 
-        logger.warning('...complete!')
+        logger.warning(f'...complete! ({count} nodes)')
```

Comment:
would also be prudent to log that we're about to drop the by-buid table

Reply:
Dont really see how it's useful, but i'll add it :)